dcm4che-core/src/main/java/org/dcm4che3/io/BulkDataDescriptor.java

```diff
@@ -102,6 +113,29 @@ public boolean isBulkData(List<ItemPointer> itemPointer, String privateCreator,
         };
     }
 
+    public static BulkDataDescriptor or(BulkDataDescriptor... descriptors) {
+        return new BulkDataDescriptor() {
+            @Override
+            public boolean isBulkData(List<ItemPointer> itemPointer, String privateCreator, int tag, VR vr, int length) {
+                for(BulkDataDescriptor bd : descriptors) {
+                    if(bd.isBulkData(itemPointer, privateCreator, tag,vr,length))
```

Comment:
Formatting: a few spaces missing here (between method arguments, and directly after "if" and for").